Resources/Prototypes/_Sunrise/Entities/Objects/Specific/Robotics/borg_tools.yml

```
@@ -484,13 +487,84 @@
+  - type: BatterySelfRecharger
+    autoRecharge: true
+    autoRechargeRate: 150
+    autoRechargePause: true
+    autoRechargePauseTime: 15
   - type: AmmoCounter
 
 - type: entity
-  name: cyborg's IAR-52
+  name: cyborg's EXP-220 Duster
+  id: WeaponLauncherDusterBorg
+  categories: [ HideSpawnMenu ]
```

YAMLlint (1.37.1)

[error] too many spaces inside brackets

(brackets)